lib/puppet/provider/cs_location/pcs.rb

```diff
@@ -87,7 +87,7 @@ def flush
     cmd = [command(:pcs), 'constraint', 'remove', @resource[:name]]
     self.class.run_command_in_cib(cmd, @resource[:cib], false)
     unless @property_hash[:node_name].nil?
-      cmd = [command(:pcs), 'constraint', 'location', 'add', @property_hash[:name], @property_hash[:primitive], @property_hash[:node_name], @property_hash[:score]]
+      cmd = [command(:pcs), 'constraint', 'location', 'add', '--force', @property_hash[:name], @property_hash[:primitive], @property_hash[:node_name], @property_hash[:score]]
```

Comment:
why is force required here?

Reply:
That's a very good question, I don't even know why that's there, tbqh.

Reply:
There's a changelog entry on 'Validation of node existence in a cluster when creating location constraints'… maybe that's it. I'll remove it for testing and have beaker check it locally.

Reply:
Yep:

```
  Error: /Stage[main]/Main/Cs_location[duncan_vip_there]: Could not evaluate: Execution of '/usr/sbin/pcs constraint location add duncan_vip_there duncan_vip debian11-64 INFINITY resource-discovery=exclusive' returned 1: Error: Node 'debian11-64' does not appear to exist in configuration, use --force to override
```